packages/devtools_app/lib/src/shared/table_data.dart

```diff
@@ -101,6 +101,11 @@ enum ColumnAlignment {
   center,
 }
 
+abstract class PinnableListEntry {
```

Comment:
should this be a Mixin instead?

Reply:
Yeah, that's what I had it as originally, but was fighting with the type check in `FlatTableState`. Done.